README.adoc

```diff
@@ -191,16 +189,12 @@ Now, open the `inventory` service from the `src/main/java/io/openliberty/guides/
 include::finish/src/main/java/io/openliberty/guides/inventory/InventoryResource.java[tags=InventoryResource]
 ----
 
-The `getPropertiesForHost()` method displays the system properties for a host name only if you set the
-`io.openliberty.guides.microprofile.inventory.inMaintenance` property to `false`. If this is the case,
-then the health of this service is `UP`.
-
-Otherwise, if the property is set to `true`, the service responds with an error message and a `503` response
-code. Thus, If the service is down for maintenance, its health is `DOWN'.
-
-Notice that the `isHealthy()` method of the `InventoryHealth` class
-works with `getPropertiesForHost()` method through `InventoryConfig` object that modifies
-the same configuration file `resources/CustomConfigSource.json`.
+The `getPropertiesForHost()` method displays the system properties for a host name only if
+the `isInMaintenance()` method called by the `InventoryConfig` object returns `false`. This happens when you set
+`io.openliberty.guides.microprofile.inventory.inMaintenance` property to `false`. Since this property is also read by
+`InventoryHealth` class through `isHealthy()` method, the health of this service is `UP` because it is
+not in maintenance. Otherwise, if the property is set to `true`, the service responds with an error message and a `503`
+response code. Thus, if the service is down for maintenance, its health is `DOWN`.
```

Comment:
What about...
The `getPropertiesForHost()` method displays the system properties for a host name only if the `isInMaintenance()` method **that is** called by the `InventoryConfig` object returns `false`. **The object returns `false`** when you set **the** `io.openliberty.guides.microprofile.inventory.inMaintenance` property to `false`. Since **the `InventoryHealth` class** also reads this property through **the** `isHealthy()` method, the health of this service is `UP` because it is not in maintenance. Otherwise, if the property is set to `true`, the service responds with an error message and a `503` response code. Thus, if the service is down for maintenance, its health is `DOWN`.

Reply:
The method returns false not the object

Reply:
We can say, "The method returns false when you set..." Would that be OK?

Reply:
yes ofcourse :D